cmd/buildah/run.go

```diff
@@ -85,9 +89,9 @@ func runCmd(c *cli.Context) error {
 		Runtime:  runtime,
 		Args:     flags,
 	}
-	runerr := builder.Run(c.Args(), options)
+	runerr := builder.Run(args, options)
 	if runerr != nil {
-		logrus.Debugf("error running %v in container: %v", c.Args(), runerr)
+		logrus.Debugf("error running %v in container: %v", args, runerr)
```

Comment:
Probably makes sense to use the container's name, after we've looked it up, since we may have been given something other than a name to start with.